src/network/network.cpp

```diff
@@ -186,13 +186,15 @@ void Network::setPendingPeersToReady() {
   }
 }
 
-dev::p2p::NodeID Network::getNodeId() { return host_->id(); }
+dev::p2p::NodeID Network::getNodeId() const { return host_->id(); }
 
-int Network::getReceivedBlocksCount() { return taraxa_capability_->getBlocks().size(); }
+int Network::getReceivedBlocksCount() const { return taraxa_capability_->getBlocks().size(); }
 
-int Network::getReceivedTransactionsCount() { return taraxa_capability_->getTransactions().size(); }
+int Network::getReceivedTransactionsCount() const { return taraxa_capability_->getTransactions().size(); }
 
-std::shared_ptr<TaraxaPeer> Network::getPeer(NodeID const &id) { return taraxa_capability_->getPeer(id); }
+std::shared_ptr<TaraxaPeer> Network::getPeer(NodeID const &id) const { return taraxa_capability_->getPeer(id); }
+
+uint64_t Network::pendingTasksNum() const { return tp_.num_pending_tasks(); }
```

Comment:
where do you use it?

Reply:
I think it was removed in develop branch and it is just left there after rebase. I will remove that